File: src/main/java/ges/resto/entity/Complement.java
package ges.resto.entity;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor

public class Complement {
    private int id;
    private String code;
    private String nom;
    private double prix;
    private ComplementType ComplementType;

    @Override
    public String toString() {
        return String.format("%03d | %-7s | %-20s | %8.2f | %-10s",
                id, code, nom, prix, ComplementType);
    }
}

File: Models/Complement.cs
using Models.Enums;

namespace Models
{
    public class Complement
    {
        public int Id { get; set; }
        public string Code { get; set; }
        public string? Photo { get; set; }
        public string Nom { get; set; }
        public decimal Prix { get; set; }
        public ComplementType ComplementType { get; set; }
        public EtatProduit Etat { get; set; }
    }
}
